Assets/Scripts/VersionFormatter.cs

```diff
@@ -23,15 +23,15 @@ public VersionFormatter(string issueRegex, string versionPattern)
 
     public string FormatBranchName(string branchName)
     {
-        var regex = new Regex(issueRegex);
+        Regex regex = new Regex(issueRegex);
```

Comment:
Use var over explicitly defining a Type when it is clear what type the right hand side is